### PR TITLE
[JBIDE-23707] clarified loading job messages in explorer

### PR DESCRIPTION
**Pull request by adietish**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by adietish** [comment]
@jeffmaury @bdshadow please review this simple improvement of job messages

**Comment by adietish** [comment]
testPR

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java [hunk 1]
```diff
@@ -99,14 +100,14 @@ void initWith(List<IProject> resources) {
...
-			startLoadJob(handler);
+			startLoadJob(getWrapped(), handler);
 			return true;
 		}
 		return false;
 	}
 
 	void startLoadJob(ProjectWrapper projectWrapper, IExceptionHandler handler) {
-		new Job("Load project contents") {
+		new Job(NLS.bind("Loading resources for project {0}...", projectWrapper.getWrapped().getName())) {
```
Comment (jeffmaury) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java]: Given the fact that the info is displayed in a general context (jobs), I would change to:
Loading resources for Openshift project...

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java [hunk 2]
```diff
@@ -132,13 +133,12 @@ protected IStatus run(IProgressMonitor monitor) {
 		}.schedule();
 	}
 
-	private void startLoadJob(IExceptionHandler handler) {
-		new Job("Load project") {
+	private void startLoadJob(IOpenShiftConnection connection, IExceptionHandler handler) {
+		new Job(NLS.bind("Loading resources for server {0}...", connection.getHost())) {
```
Comment (jeffmaury) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java]: Idem see my previous comment

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java (hunk 1 above)
Comment (adietish) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/ConnectionWrapper.java]: +1,  but given the limited space there is to display job messages in the status bar, I'll shorten the message.

**Comment by adietish** [comment]
replaced with #1386 since this PR was requesting origin/topics-branch to be merged whereas it should have been myfork/topics-branch

**Pull request closed without merging**